pkg/cmd/github-pull-request-make/main.go

```diff
@@ -105,25 +139,30 @@ func pkgsFromDiff(r io.Reader) (map[string]pkg, error) {
 		switch {
 		case bytes.HasPrefix(line, []byte(newFilePrefix)):
 			curPkgName = filepath.Dir(string(bytes.TrimPrefix(line, []byte(newFilePrefix))))
-		case newGoTestRE.Match(line):
-			curPkg := pkgs[curPkgName]
-			curPkg.tests = append(curPkg.tests, string(newGoTestRE.ReplaceAll(line, []byte(replacement))))
-			pkgs[curPkgName] = curPkg
-		case currentGoTestRE.Match(line):
-			curTestName = ""
+			isGoPackage = bytes.HasSuffix(line, []byte(goFileSuffix))
+		case currentGoTestRE.Match(line) && isGoPackage:
+			curTestName := ""
 			if !bytes.HasPrefix(line, []byte{'-'}) {
 				curTestName = string(currentGoTestRE.ReplaceAll(line, []byte(replacement)))
 			}
-		case bytes.HasPrefix(line, []byte{'-'}) && (bytes.Contains(line, []byte(".Skip")) || bytes.Contains(line, []byte("skip."))):
-			if curPkgName != "" && len(curTestName) > 0 {
+			if curPkgName != "" && len(curTestName) > 0 && okToStress(curTestName) {
 				curPkg := pkgs[curPkgName]
-				curPkg.tests = append(curPkg.tests, curTestName)
+				curPkg.maybeAddTest(curTestName)
 				pkgs[curPkgName] = curPkg
 			}
 		}
 	}
 }
 
+func okToStress(testName string) bool {
+	for _, dontStress := range dontStressTests {
+		if dontStress.Match([]byte(testName)) {
```

Comment:
Nit: Can use `MatchString` to avoid the `[]byte` cast

Reply:
Done.